mne/selection.py

```diff
@@ -114,3 +122,75 @@ def read_selection(name, fname=None, info=None, verbose=None):
     if spacing == 'new':  # "new" or "old" by now, "old" is default
         sel = [s.replace('MEG ', 'MEG') for s in sel]
     return sel
+
+
+def _divide_to_regions(raw, add_stim=True):
```

Comment:
We shouldn't exclude e.g. EEG channels, channels of other types. Instead, we should probably make "additional selections" on the fly based on the `n_channels` parameter of `plot_raw`, e.g. for `n_channels=20` you'd get 3 "selections" of EEG (just in their order), followed by the "selection" of stim channels (assuming those were next in the file), etc.